pkg/accountdb/dbx/accountdb.dbx

```diff
@@ -0,0 +1,35 @@
+// dbx.v1 golang accountdb.dbx .
+
+model user (
+    key id
+    unique email
```

Comment:
Do we normalize emails? e.g. `first.last+1@example.com` and `firstlast@example.com` might refer to the same person.